src/dynarec/arm64/dynarec_arm64_f30f.c

```diff
@@ -334,19 +334,15 @@ uintptr_t dynarec64_F30F(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int n
             GETGX(v0, 1);
             u8 = F8;
             d0 = fpu_get_scratch(dyn);
-            if(u8==0b00000000 || u8==0b01010101 || u8==0b10101010 || u8==0b11111111) {
-                VDUP_16(d0, v1, u8&3);

```

Comment:
Maybe it needs a `VMOVeD(d0, 0, v1, 1);` before than?

Reply:
Ah we can do that?

Reply:
or a `VDUPQ_16(d0, v1, u8&3+4);`  directly?

Reply:
The Q means high part?

Reply:
But it should be `(u8&3) + 4)` IIUC, yes?